recipe/build-mpi.sh

```diff
@@ -151,23 +146,32 @@ export LIBRARY_PATH="$PREFIX/lib"
             --with-wrapper-fcflags="-I$PREFIX/include" \
             --with-wrapper-ldflags="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib" \
             --with-sge \
-            $build_with_cuda || (cat config.log; false)
+            --with-hwloc=$PREFIX \
+            --with-libevent=$PREFIX \
+            --with-zlib=$PREFIX \
+            --enable-mca-dso \
```

Comment:
Q: We don't need `=yes`?

Reply:
I don't think so. I'm testing this locally right now, also trying how `--with-cuda` works.